lisa/core/customScript.py

```diff
@@ -0,0 +1,98 @@
+from __future__ import annotations
+
+from hashlib import md5
+from pathlib import Path
+from typing import TYPE_CHECKING, List, Optional, Type
+
+from lisa.core.tool import Tool
+from lisa.util.executableResult import ExecutableResult
+from lisa.util.process import Process
+
+if TYPE_CHECKING:
+    from lisa.core.node import Node
+
+
+class CustomScript:
```

Comment:
Also, for what it's worth, I know people asked for this kind of support in the "code review" meeting (which was more of a design review 😅), and I think for now we can merge it, but I don't want us to be stuck maintaining this as we've hardly thought about its implications and costs.

Reply:
For instance, I think multiple ideas were thrown out. For one, we should make it easy to call an arbitrary command line tool, like `class Bash(Tool) ... bash.run(<whatever CLI tool>)`. We may want to make it easy to upload and run a "custom" script, but there's actually no real difference between a custom script and any other binary, so that makes we wonder if we need this in the first place.

Reply:
I'm thinking about to merge custom script back to commands. Actually, I used Tool class at beginning, but remove it later. I will change it to use Tools flow back.